src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs

```diff
@@ -74,6 +74,9 @@ public static bool IsWindowsIoTCore
         public static bool IsWindows7 => GetWindowsVersion() == 6 && GetWindowsMinorVersion() == 1;
         public static bool IsWindows8x => GetWindowsVersion() == 6 && (GetWindowsMinorVersion() == 2 || GetWindowsMinorVersion() == 3);
 
+        public static string LibcRelease => string.Empty;
```

Comment:
I meant to use "glibc_not_found" in both cases. This string goes to telemetry and changing it would mess up the data. It's ok to change for Netfx as its not used for that

Reply:
Sry i read the comment wrong, i will correct it